data.sql

```diff
@@ -10,4 +10,25 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005-06-12', 1, true, 45);
```

Comment:
- [ Required ] I think it would be nice if the data you enter in the `weight_kg` column for the first four records match the data provided in the project requirements. Currently, the values you are entering in the `weight_kg` column are `positive` while the values provided in the project requirements are `negative`. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-11-09 08-34-22](https://github.com/FilbertKimasa/Vet-clinic-database/assets/16392046/cec2059b-b37a-427d-a93e-db71689ba5c6)